av_car_description/urdf/mondeo_dca/mondeo_dca_sensors.xacro

```diff
@@ -12,50 +12,146 @@
     <origin xyz="-0.11015 0.3 -0.06242" rpy="0 0.087266 0"/>
   </xacro:camera_mount>
 
+  <xacro:camera_mount name="fsp_l_sensor" parent="camera_fsp_l_mount">
```

Comment:
Shall we keep it consitent with other camera frames and we use  `camera_` prefix for `mount` and `optical`?